### ISSUE TITLE
Add MIT license?

### ISSUE THREAD

**Issue by codykingham** [Copenhagen-Alliance/versification-specification#12]
This is a great resource. I assume it is all open source? Why not add a friendly MIT license? :)

**Comment by jonathanrobie** [comment]
Yes, it is meant to be open source.  Here are the bits I think we need:

Copyright holder:  Copenhagen Alliance
We must attribute the upstream Tyndale House spreadsheet
We want users to attribute us as the source for our data and code
Code must be covered by a code license
Specification and Data must be covered by an appropriate license like Creative Commons

**Comment by jsarabia** [comment]
I realize this is a two and a half year old issue, but I'm curious what the status of this issue is.

If it is still intended to be open source, who should be contacted in regards to uploading the respective licenses?

**Comment by jcuenod** [comment]
@jonathanrobie I'm needing a license to use this as well...
